### PR TITLE
Questions protected route

### PR DESCRIPTION
**Pull request by JacobCrozier**
now /questions will redirect to login if user not logged in 

logout feature implemented

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,35 +9,49 @@ import './css/Aua.css';
...
+                    />
+                    <Route
+                        path='/questions'
+                        render={props =>
+                            this.auth.isAuthenticated() ? (
+                                <MainPanel auth={this.auth} {...props} />
+                            ) : (
+                                <Redirect to='/login' />
+                            )
+                        }
```
Comment (j-blackmore) [inline src/App.js]: fantastic work

File: src/components/MainPage/AndBanner.js [hunk 2]
```diff
@@ -13,55 +13,22 @@ import SearchIcon from '@material-ui/icons/Search';
...
-            //     <Link to='/questions'>Questions</Link>
-            //     <Button
-            //         onclick={isAuthenticated() ? logout : login}
-            //         variant='outlined'
-            //         color=''
-            //         className=''
-            //     >
-            //         {isAuthenticated() ? 'Log Out' : 'Login'}
-            //     </Button>
-            // </AppBar>
```
Comment (j-blackmore) [inline src/components/MainPage/AndBanner.js]: lovely clean up

File: src/components/landing-page/Login.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+
+import { makeStyles } from '@material-ui/core/styles';
+import PostButton from '../Question/PostButton';
```
Comment (j-blackmore) [inline src/components/landing-page/Login.js]: can this be lowercase question please

File: src/App.js (hunk 1 above)
Comment (JacobCrozier) [reply src/App.js]: i thought so too

File: src/components/landing-page/Login.js (hunk 3 above)
Comment (JacobCrozier) [reply src/components/landing-page/Login.js]: will do now

**Comment by j-blackmore** [comment]
LGTM

**Pull request merged**